added pytorch implementation of UNet and UNet++

ris27hav opened the pull request.

ris27hav committed:
Create readme.md

ris27hav committed:
Add files via upload

ris27hav committed:
Add files via upload

ris27hav committed:
Add files via upload

ris27hav committed:
Delete result2.png

ris27hav committed:
Add files via upload

ris27hav committed:
Add files via upload

ris27hav committed:
Add files via upload

ris27hav committed:
Update README.md

ris27hav committed:
Update README.md

ris27hav committed:
Update dataloader.py

ris27hav committed:
Update train.py

ris27hav committed:
Update main.py

ris27hav committed:
Update utils.py

ris27hav committed:
Create readme.md

ris27hav committed:
add images

ris27hav committed:
Delete readme.md

ris27hav committed:
added scripts for UNet model

ris27hav committed:
Update README.md

ris27hav committed:
Update README.md

ris27hav committed:
Delete results2.png

namangup commented:
Change the parent folder name to semantic_segmentation, format code with [black](https://github.com/psf/black). Other than that LGTM

ris27hav committed:
update all the files

fixed the mistakes,
added result of training UNet,
added minor but imp changes to models

ris27hav committed:
remove .vscode + fix readme

ris27hav commented:
I have done the required changes and added my training results as well. Perhaps this should do.

ris27hav committed:
Update README.md

ris27hav committed:
Merge branch 'pclubiitk:master' into master
